src/components/ValidateCodeActionModal/ValidateCodeForm/BaseValidateCodeForm.tsx

```diff
@@ -116,9 +117,15 @@ function BaseValidateCodeForm({
             if (focusTimeoutRef.current) {
                 clearTimeout(focusTimeoutRef.current);
             }
-            focusTimeoutRef.current = setTimeout(() => {
+
+            if (!Browser.isMobileSafari()) {
+                focusTimeoutRef.current = setTimeout(() => {
+                    inputValidateCodeRef.current?.focusLastSelected();
+                }, CONST.ANIMATED_TRANSITION);
+            } else {
                 inputValidateCodeRef.current?.focusLastSelected();
-            }, CONST.ANIMATED_TRANSITION);
+            }
```

Comment:
Mobile safari can't auto focus with delay, otherwise the keyboard won't show.
ref: https://github.com/Expensify/App/issues/10414

Reply:
Thanks for this comment. Can you put it as a code comment in the code?

Reply:
Done